2021-11-11 14:28:20,029 INFO    MainThread:66108 [wandb_setup.py:_flush():71] setting env: {}
2021-11-11 14:28:20,029 INFO    MainThread:66108 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-11 14:28:20,029 INFO    MainThread:66108 [wandb_init.py:_log_setup():357] Logging user logs to /home/joy/projects/music-controllable-diffusion/models/gmiw_vae/wandb/run-20211111_142820-txpr273t/logs/debug.log
2021-11-11 14:28:20,029 INFO    MainThread:66108 [wandb_init.py:_log_setup():358] Logging internal logs to /home/joy/projects/music-controllable-diffusion/models/gmiw_vae/wandb/run-20211111_142820-txpr273t/logs/debug-internal.log
2021-11-11 14:28:20,030 INFO    MainThread:66108 [wandb_init.py:init():390] calling init triggers
2021-11-11 14:28:20,030 INFO    MainThread:66108 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-11 14:28:20,030 INFO    MainThread:66108 [wandb_init.py:init():435] starting backend
2021-11-11 14:28:20,030 INFO    MainThread:66108 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-11 14:28:20,044 INFO    MainThread:66108 [backend.py:ensure_launched():199] starting backend process...
2021-11-11 14:28:20,564 INFO    MainThread:66108 [backend.py:ensure_launched():204] started backend process with pid: 66211
2021-11-11 14:28:20,565 INFO    MainThread:66108 [wandb_init.py:init():444] backend started and connected
2021-11-11 14:28:20,567 INFO    MainThread:66108 [wandb_init.py:init():503] updated telemetry
2021-11-11 14:28:20,591 INFO    MainThread:66108 [wandb_init.py:init():533] communicating current version
2021-11-11 14:28:21,937 INFO    MainThread:66108 [wandb_init.py:init():538] got version response 
2021-11-11 14:28:21,937 INFO    MainThread:66108 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-11 14:28:22,102 INFO    MainThread:66108 [wandb_init.py:init():576] starting run threads in backend
2021-11-11 14:28:22,791 WARNING MainThread:66108 [wandb_init.py:init():891] interrupted
Traceback (most recent call last):
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 870, in init
    run = wi.init()
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 581, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 216, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 446, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 232, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/joy/.venv/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 32, in get
    is_set = self._object_ready.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
